### ISSUE TITLE
NotFoundError (key) when restoring from a checkpoint

### ISSUE THREAD

**Issue by ghost** [DeepSceneSeg/AdapNet-pp#6]
I've tried to restore the model from SUN RGB checkpoint. Yet, I've received an error:

`> NotFoundError (see above for traceback): Key resnet_v2_50/block1/unit_1/bottleneck_v1/conv1/BatchNorm/beta not found in checkpoint`

` OP_REQUIRES failed at save_restore_v2_ops.cc:184 : Not found: Key resnet_v2_50/block1/unit_1/bottleneck_v1/conv1/BatchNorm/beta not found in checkpoint
`

Here's my config:
```
gpu_id: '0'
model: 'SSMA'
num_classes: 38
checkpoint: '/home/user_foo/Downloads/sun_ssma_depth/model'
test_data: '/home/user_foo/workspace/ssma/ssma_source/dataset/tf_records_name'
batch_size: 5
skip_step: 1
height: 384
width: 768
```

I haven't received such an error with any other pretrained models.

**Comment by avalada** [comment]
Are you using the evaluate.py in the Adapnet_pp repo to run SSMA evaluation? You need to run evaluate.py from the SSMA repo for it.
evaluate.py in Adapnet_pp repo is specifically for running the unimodal models.

**Comment by ghost** [comment]
Yes, as I've explaines before, it works fine with Scannetv2 pretrained
model (only changing classes number and the model dir in the config).

**Comment by avalada** [comment]
We are not able to reproduce this issue. Can you checkout a fresh copy of the repository and try it. Your config says SSMA so please post issues that are only related to AdapNet++ in this repo.

Note: If you are trying to use the evaluate.py for the multimodal model then you need to use this script https://github.com/DeepSceneSeg/SSMA/blob/master/evaluate.py
and not
https://github.com/DeepSceneSeg/AdapNet-pp/blob/master/evaluate.py
